packages/resource/package.json

```diff
@@ -57,7 +57,6 @@
   "devDependencies": {
     "arrify": "^1.0.0",
     "async": "^2.1.2",
-    "google-auto-auth": "^0.5.2",
```

Comment:
Still need this! https://github.com/ofrobots/google-cloud-node/blob/aa5f008e1ebf8106877e23503b4ae508d20a86f7/packages/resource/system-test/resource.js#L22

Although, that can probably be re-written in the future at some point.